### PR TITLE
Redoing stuff from October

### PR DESCRIPTION
**Pull request by LawrenceHecht**
See #338 (https://github.com/chaoss/wg-diversity-inclusion/pull/338)

### CODE REVIEW

**Comment by germonprez** [comment]
Thanks @LawrenceHecht 

Taking a look, I think your additions are much appreciated. From a formatting perspective, we follow the template from here: 
https://github.com/chaoss/metrics/blob/master/resources/metrics-template.md

This would have an impact on a few of the header changes that you suggest. 

@Ruth-ikegah -- what do you think of the proposed changes?

File: focus-areas/project-and-community/project-burnout.md [hunk 1]
```diff
@@ -24,37 +24,42 @@ Project Burnout should be understood within projects and managed effectively to
...
+  
+* On average, are you more likely to feel energized or emotionally drained after engaging with the project's community?
+  * Energized
+  * Emotionally drained
+
+
+_This question is currently phrased broadly to identify a) if contributions will increase or decrease and b) why the changes will occur. The Working Group should discuss what the goals are. Then we can operationalize and provide sample questions. In other words, we can just look at plans to decrease contributions based on increased commitments._
+
+
+* In the next six months, how likely are you to increase or decrease the time you contribute to the project for the following reasons: (Very likely to increase; Somewhat likely to increase; Neutral—no change; Somewhat likely to decrease; Very likely to decrease; Don’t know)
```
Comment (jwflory) [inline focus-areas/project-and-community/project-burnout.md:44]: I feel like asking someone to predict where their life will be in six months, either in personal or work commitments, is a really hard ask! 2020 really made this clear to me, especially with COVID-19 and changes to care-taking that can happen very abruptly with little advance warning.

**Comment by LawrenceHecht** [comment]
Signed-off-by @LawrenceHecht

**Comment by LawrenceHecht** [comment]
signed-off-by Lawrence Hecht LawrenceHecht@users.noreply.github.com

**Comment by LawrenceHecht** [comment]
> Taking a look, I think your additions are much appreciated. From a formatting perspective, we follow the template from here:
> https://github.com/chaoss/metrics/blob/master/resources/metrics-template.md
> This would have an impact on a few of the header changes that you suggest.

@germonprez I tried to tweak a couple of headers -- but I'm still having problems with the DCO through my browser, so my update didn't go through. If that still doesn't work, then just get rid of the "###" tags in this document -- they are not needed.

File: focus-areas/project-and-community/project-burnout.md (hunk 1 above)
Comment (LawrenceHecht) [reply focus-areas/project-and-community/project-burnout.md:44]: @jwflory We can make the question backwards looking and about the last 12 months.

**Comment by LawrenceHecht** [comment]
signed-off-by Lawrence Hecht LawrenceHecht@users.noreply.github.com

File: focus-areas/project-and-community/project-burnout.md (hunk 1 above)
Comment (ElizabethN) [reply focus-areas/project-and-community/project-burnout.md:44]: We are capturing these comments for rediscussion during the metric refresh.

**Comment by ElizabethN** [comment]
@LawrenceHecht these changes have been captured in another doc and will be resurfaced when we do our periodic review of the Burnout metric. This should be happening in the next month or so. :+1: I'm closing this for now. Thanks for your work here! :pray:

**Pull request closed without merging**